artifacts_test.py

```diff
@@ -0,0 +1,71 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+import os
+import time
+
+from sdcm.tester import ClusterTester
+from sdcm.cluster import get_username
+from sdcm.utils.common import format_timestamp
+from sdcm.sct_events import get_logger_event_summary
+
+
+STRESS_CMD: str = "/usr/bin/cassandra-stress"
+
+
+class ArtifactsTest(ClusterTester):
+    @property
+    def node(self):
+        return self.db_cluster.nodes[0]
+
+    def restart_scylla(self):
+        self.db_cluster.restart_scylla()
+
+    def run_cassandra_stress(self, args: str):
+        result = self.node.remoter.run(f"{STRESS_CMD} {args} -node {self.node.ip_address}")
+        assert "java.io.IOException" not in result.stdout
+        assert "java.io.IOException" not in result.stderr
+
+    def check_scylla(self):
+        self.node.run_nodetool("status")
+        self.run_cassandra_stress("write n=10000 -mode cql3 native -pop seq=1..10000")
+        self.run_cassandra_stress("mixed duration=1m -mode cql3 native -rate threads=10 -pop seq=1..10000")
+
+    def test_scylla_service(self):
+        with self.subTest("check Scylla server after installation"):
+            self.check_scylla()
+
+        with self.subTest("check Scylla server after stop/start"):
+            self.restart_scylla()
```

Comment:
The restart_scylla() is stopping, and then restart.

Original artifact-tests contain 3 subtests, one lost subtest is restart scylla-server by `systemctl restart scylla-server`

Reply:
I was not sure about this test, but I've added this test anyway.
Done.